### ISSUE TITLE
Broken: Symlinking to install

### ISSUE THREAD

**Issue by springmeyer** [mapbox/mason#69]
The documented installation instructions (https://github.com/mapbox/mason#installation) actually don't work. You'll get something like:

```sh
~/tmp 》mason install libzip 0.11.2
/usr/local/bin/mason: line 54: /usr/local/bin/mason.sh: No such file or directory
```

I presume this broke in 
https://github.com/mapbox/mason/commit/ad3dd780fbfe209042b5ebe96c2e74dcb7659052#diff-5c29c2e513aadfe372fd0af7553b5a6c and test coverage for this is missing since we don't symlink on travis and also set MASON_DIR: https://github.com/mapbox/mason/blob/master/.travis.yml#L20.

**Comment by rclark** [comment]
How out of the question would it be to remove mason's dependence on living in `~/.mason` entirely?

I'm considering how I might include mason as a dependency in a `package.json` file, and how I might utilize it in something like a `postinstall` script to load up binary dependencies.

**Comment by jfirebaugh** [comment]
Related to #140 and #141.

**Comment by TobWen** [comment]
You should remove the installation howto from https://github.com/mapbox/mason since it also contain the linkage. With the current bash script to detect the mason's root directory, many users will run into the same problem again and again.

**Comment by springmeyer** [comment]
Fixed with #198
